adhocracy4/test/helpers.py

```diff
@@ -33,3 +37,29 @@ def redirect_target(response):
 def render_template(string, context=None):
     context = Context(context or {})
     return Template(string).render(context)
+
+
+@contextmanager
+def skip_background_mail():
```

Comment:
> 1. Move the entire thing as it is from helpers to the email test package, because it is only useful there

i disagree as the skip_background_task context is required to test if emails are/would sent in other modules. see for example the `test_report_api.py` [1]
because i expect this to be a common pattern i think it is nice to have a context that is named explicitly and does not require further knowledge about the the EMail implementation. 

Another option would be to globally skip background tasks for all tests and just enable them if they are explicitly needed. Although i am not entirely sure this is doable.

[1] https://github.com/liqd/adhocracy4/pull/113/files#diff-460ed8a9f44094f159330c5b76202e04R56